pkg/catalog/template.go

```diff
@@ -37,7 +36,7 @@ const MacroPackageProject = `
 
 const NewPackageTemplate = `
 {{/* Automatically generated by https://github.com/anz-bank/sysl-catalog it is strongly recommended not to edit this file */}}
-[Back](../README.md) | [Chat with us]({{ChatLink}}) | [New bug or feature request]({{FeedbackLink}})
+[Back](../README.md)
```

Comment:
Should we put the default public contact links here?
https://anzoss.slack.com/messages/sysl-catalog/ and
https://github.com/anz-bank/sysl-catalog/issues/new

Reply:
I think this should just be in the readme

Reply:
Sure